assets/scss/_custom.scss

```diff
@@ -96,6 +96,26 @@ body.td-home main[role="main"] > section:first-of-type .content p:first-child {
   }
 }
 
+.section-feature#kubeweekly {
+
+  .kubeweekly-signup {
+    color: #3371E3;
+    font-weight: 400;
+    font-size: 1.8rem;
+  }
+  > .kubeweekly-inner {
+    form p {
+      text-align: center;
+      font-size: 1.2rem;
+      color: $primary;
+    }
+    margin-left: auto;
+    margin-right: auto;
+    width: 100vw;
```

Comment:
This creates unnecessary horizontal scrolling for the page in Chrome/Linux. If we change the value to `100%`, everything is fine. However, just removing the whole line is good enough as well (I've tested it in Firefox and Chrome, both in Linux).
```suggestion
```